### ISSUE TITLE
PlaceMarketOrderAsync: Exception Thrown, invalid price 0

### ISSUE THREAD

**Issue by Lumaswell** [dougdellolio/coinbasepro-csharp#219]
I've been using the PlaceLimitOrderAsync for a while now, and recently added some functions for the PlaceMarketOrderAsync to my code.  The first time I tried to use it (and subsequently thereafter) I received an exception:

'CoinbasePro.Exceptions.CoinbaseProHttpException' in mscorlib.dll
Msg - Invalid price 0

Since there's no input for price on the function itself (unless I've grievously misread something) I think I found a bug.

**Comment by Lumaswell** [comment]
In looking back at the function I did not specify the MarketOrderAmountType setting, I have set that now and will respond back if it works.

**Comment by Lumaswell** [comment]
Unfortunately I have to confirm the bug, both on the live server and the sandbox.   I updated all my add ons and packages just to be safe, up to date as of 1/29/2020. The code snippet I used for testing follows:
```
var Sellresponse = await CurrentClient.OrdersService.PlaceMarketOrderAsync(OrderSide.Sell, ProductType.BtcUsd, 1.0m, MarketOrderAmountType.Size);
```

**Comment by dougdellolio** [comment]
hi @Lumaswell. This does look like a bug. I have fixed it here (https://github.com/dougdellolio/coinbasepro-csharp/commit/1c707c85d1e721ac8ac977464bb210cc5b070878) and released a new version, please update to v1.0.34 and let me know how it looks. thanks for reporting this!

**Comment by Lumaswell** [comment]
Awesome, thanks for the quick fix.  Worked great in the sandbox, will let you know if there are any further issues, but I'm not expecting any.
